Example/AztecUITests/HighPriorityIssuesTests.swift

```diff
@@ -0,0 +1,102 @@
+//
+//  HighPriorityIssuesTests.swift
+//  AztecUITests
+//
+//  Created by brbrr on 11/10/17.
+//  Copyright © 2017 Automattic Inc. All rights reserved.
+//
+
+import XCTest
+
+class HighPriorityIssuesTests: XCTestCase {
+        
+    private var app: XCUIApplication!
+    private var richTextField: XCUIElement!
+    
+    override func setUp() {
+        super.setUp()
+        
+        // In UI tests it is usually best to stop immediately when a failure occurs.
+        continueAfterFailure = false
+        // UI tests must launch the application that they test. Doing this in setup will make sure it happens for each test method.
+        XCUIDevice.shared().orientation = .portrait
+        app = XCUIApplication()
+        app.launch()
+        
+        let tablesQuery = app.tables
+        tablesQuery.staticTexts[elementStringIDs.emptyDemo].tap()
+        
+        let richTextField = app.textViews[elementStringIDs.richTextField]
+        richTextField.tap()
+    }
+    
+    override func tearDown() {
+        // Put teardown code here. This method is called after the invocation of each test method in the class.
+        super.tearDown()
+    }
+    
+    // Gihtub issue #393
```

Comment:
I'd go ahead and add the full link to the issue here, to make it easier to find. (Also, just a small typo in "Gihtub")

Reply:
✅